.github/workflows/nightly-aliyunlinux-sgx2.yml

```diff
@@ -104,7 +104,15 @@ jobs:
       # occlum-pal should be installed manually
       - name: Install Occlum stack
         run: |
+          pushd ${WORK_DIR}
+          if [ ! $(lsmod | grep enable_rdfsbase) ]; then
+            wget https://github.com/occlum/enable_rdfsbase/archive/master.zip -O enable_rdfsbase.zip && unzip enable_rdfsbase.zip
+            pushd enable_rdfsbase-master
```

Comment:
Use `yum install occlum-rdfsbase-dkms` to install enable_rdfsbase in sgx2 alinux machine